src/components/index/sections/Jumbotron.tsx

```diff
@@ -45,25 +45,33 @@ const useStyles = makeStyles((theme) =>
       color: theme.palette.common.white,
       border: `2px solid ${theme.palette.primary.main}`,
       borderRadius: theme.spacing(3),
-      padding: theme.spacing(1.5, 5),
       fontWeight: 500,
       fontSize: theme.typography.pxToRem(15),
       width: theme.spacing(27),
+      height: theme.spacing(7),
       margin: theme.spacing(2),
+      [theme.breakpoints.up('sm')]: {
+        margin: theme.spacing(3),
+      },
     },
     podkrepiButton: {
       color: theme.palette.common.white,
       backgroundColor: theme.palette.primary.main,
       border: `2px solid ${theme.palette.primary.main}`,
       borderRadius: theme.spacing(3),
-      padding: theme.spacing(1.5, 5),
       fontWeight: 500,
       fontSize: theme.typography.pxToRem(15),
+      width: theme.spacing(27),
```

Comment:
Adding fixed width is not the best way to handle the buttons as they will wrap if the text is changed.
Maybe we can replace it with `minWidth` of that's the visual expectation.